tools/eggnog_mapper/eggnog_mapper/eggnog_mapper.xml

```diff
@@ -42,7 +42,9 @@
     <inputs>
         <param name="input" type="data" format="fasta" label="Fasta sequences to annotate"/>
         <param name="eggnog_data" type="select" label="Version of eggNOG Database">
-            <options from_data_table="eggnog_mapper_db"/>
+            <options from_data_table="eggnog_mapper_db">
```

Comment:
We probably need to rename the data table to `eggnog_mapper_db_versioned` and the file.